### ISSUE TITLE
Dark mode option doesn't appear to work for set_theme service

### ISSUE THREAD

**Issue by realroywalker** [thomasloven/hass-browser_mod#338]
My Home Assistant version: 2022.6.7

**What I am doing:**
I am attempting to set the theme dark mode to change between light/dark mode using the browser_mod.set_theme service call

**What I expected to happen:**
When the service is called with 'dark: true' dark mode should be selected, when called with 'dark: false' it should select 'light' mode for the theme.

**What happened instead:**
Nothing appears to happen to the dark/light/auto mode - the named theme is however applied correctly.

**Minimal** steps to reproduce:

```yaml
# The least amount of code or steps possible to reproduce my error

service: browser_mod.set_theme
data:
  theme: default
  dark: false
```

**Error messages from the browser console:**

```
CARD-MOD 3.1.5 IS INSTALLED
browser_mod.js:145 BROWSER_MOD 1.5.3 IS INSTALLED    DeviceID: ea2446e5-bb962389
(no additional messages appear when calling the service)
```

---

**By replacing the space in the checkboxes ([ ]) with an X below, I indicate that I:**

- [X] Understand that this is a channel for reporting bugs, not a support forum (https://community.home-assistant.io/).

- [X] Have made sure I am using the latest version of the plugin.

- [X] Have followed the troubleshooting steps of the "Common Problems" section of https://github.com/thomasloven/hass-config/wiki/Lovelace-Plugins.

- [X] Understand that leaving one or more boxes unticked or failure to follow the template above may increase the time required to handle my bug-report, or cause it to be closed without further action.

**Comment by veedubb65** [comment]
This affects me as well. Currently running Home Assistant 2022.7.3 and Browser_Mod 5c7f68c.

The piece of script I'm running is:

```
action:
  - service: browser_mod.set_theme
    data:
      theme: Fluent Purple
      dark: false
```



`dark: true` fails as well.

**Comment by DJaeger** [comment]
I have the same issue with

```action:
  - service: browser_mod.set_theme
    data:
      theme: Mushroom
      dark: true
```
Tried with different themes, that support light and dark mode.
Tried from different browsers with different browsers accessing browser_mod.

Versions:
```
browser_mod 1.5.3
Home Assistant 2022.7.5
Supervisor 2022.07.0
Operating System 8.2
Frontend 20220707.1 - latest
```

**Comment by thomasloven** [comment]
Closed because of Browser Mod 2.0 release.
Please see #354.
